params.go

```diff
@@ -124,6 +124,10 @@ type Params struct {
 	// Account is deprecated form of StripeAccount that will do the same thing.
 	// Please use StripeAccount instead.
 	Account string
+
+	// This may be used to provide a new header or override an existing one on
+	// the HTTP request.
+	Headers map[string]string
```

Comment:
Great comments!
1) Switched to `net/http.Header`
2) `http.Header` doesn't automatically initialize on `Add`, so I added a check in `ephemeralkey.New` to initialize it.
3) `golint` actually didn't seem to catch this (probably because it's not a top-level export), but fixed anyway (and a few other things `golint` turned up)